<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="10625" started-at="2017-12-17T01:28:21Z" finished-at="2017-12-17T01:28:32Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="10625" started-at="2017-12-17T01:28:21Z" finished-at="2017-12-17T01:28:32Z">
      <class name="radiocheckdropdown.DropDownTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:radiocheckdropdown.DropDownTest@3532ec19]" name="setUp" is-config="true" duration-ms="2076" started-at="2017-12-16T20:28:21Z" finished-at="2017-12-16T20:28:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="dropDownTest()[pri:0, instance:radiocheckdropdown.DropDownTest@3532ec19]" name="dropDownTest" duration-ms="8523" started-at="2017-12-16T20:28:24Z" finished-at="2017-12-16T20:28:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Option 2] but found [Option 1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Option 2] but found [Option 1]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at radiocheckdropdown.DropDownTest.dropDownTest(DropDownTest.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dropDownTest -->
      </class> <!-- radiocheckdropdown.DropDownTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
